site/content/docs/howto/configure-supervisor-with-workspace_one_access.md

```diff
@@ -0,0 +1,153 @@
+   1. Enter the Redirect URL. This is the `spec.issuer` you configured in your `FederationDomain` appended with `/callback`.
+   1. Enter the Client ID to a value such as "pinniped-supervisor" (this cannot be changed later).
+   1. Enter the Client Secret. This should be set to secure value such as the output of `openssl rand -hex 32`.
+   1. Set both "Open in Workspace ONE Web" and "Show in User Portal" options to "No"
+   1. Set the desired Access Policies for the app, such as requiring smart card login.
+   1. Save and assign the app to the desired users and/or groups. This can be used to restrict which users can log in to Kubernetes using this integration.
+1. Configure the token TTLs and scopes. Navigate to _Catalog_ > _Settings_ > _Remote App access_ and click on the "pinniped-supervisor" client.
+   1. Edit the _Client Configuration_:
+      1. `Issue Refresh Token` must be checked
+      1. Set `Access Token Time-To-Live (TTL)` to 5 minutes
+      1. Set `Refresh Token Time-To-Live (TTL)` to 9 hours (or shorter if you wish to require more frequent logins)
+      1. Set `Idle Token Time-to-Live (TTL)` to 9 hours (or shorter if you wish to enforce an inactivity timeout)
+   1. Edit the _Scope_ configuration:
+      1. `OpenID` must be checked
+      1. Check `Email` if you plan to use email as the username claim
+      1. Check `Group` if you plan to use groups in your Kubernetes environment
+      1. Uncheck all other scopes
+
+## Configure the Supervisor
+
+Create an [OIDCIdentityProvider](https://github.com/vmware-tanzu/pinniped/blob/main/generated/{{< latestcodegenversion >}}/README.adoc#oidcidentityprovider) in the same namespace as the Supervisor.
+
+For example, this OIDCIdentityProvider and corresponding Secret use Workspace ONE Access's `email` claim as the Kubernetes username:
+
+```yaml
+apiVersion: idp.supervisor.pinniped.dev/v1alpha1
+kind: OIDCIdentityProvider
+metadata:
+  namespace: pinniped-supervisor
+  name: ws1
+spec:
+
+  # Specify the issuer URL (no trailing slash). Change this to be the
+  # actual issuer of your Workspace ONE Access environment.  Note that
+  # the Workspace ONE Access issuer ends with the string "/SAAS/auth."
+  issuer: https://ws1.my-company.com/SAAS/auth
+
+  # Specify how to form authorization requests to Workspace ONE Access.
+  authorizationConfig:
+
+    # Request any scopes other than "openid" for claims besides
+    # the default claims in your token. The "openid" scope is always
+    # included.
+    #
+    # See the example claims below to learn how to customize the claims returned.
+    additionalScopes: [group, email]
+
+  # Specify how Workspace ONE Access claims are mapped to Kubernetes identities.
+  claims:
+
+    # Specify the name of the claim in your Workspace ONE Access token that will
+    # be mapped to the "username" claim in the tokens minted by the Supervisor.
+    #
+    # The sub claim is recommended for environments that want
+    # to use a more stable identifier.
```

Comment:
```suggestion
    # User's emails can change. Use the sub claim if 
    # your environment requires a stable identifier.
```
Trying to make it more clear why the email is not suitable and what it means to be stable.